packages/wallets/src/i18n.ts

```diff
@@ -0,0 +1,40 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+const DEFAULT_LANGUAGE = 'EN';
+let currentLanguage = localStorage.getItem('i18n_language') || DEFAULT_LANGUAGE;
```

Comment:
how about BE language like in get settings ?

Reply:
We still rely on core for now, so its fine. The thing that we need to consider for now is if we change language from language switcher we need to listen to the localStorage change and remove/change the language in url param